R/backend-mssql.R

```diff
@@ -350,6 +350,16 @@ simulate_mssql <- function(version = "15.0") {
           sql_expr(DATEPART(QUARTER, !!x))
         }
       },
+
+      # clock ---------------------------------------------------------------
+      add_days = function(x, n, ...) {
+        rlang::check_dots_empty(...)
```

Comment:
You can omit the rlang namespace. The rlang package is so commonly used that it is usually imported.
And you either do `check_dots_empty()` or `check_dots_empty0(...)` but you must not pass `...` to `check_dots_empty()`.

```suggestion
        check_dots_empty()
```